src/commands/upload_file.ts

```diff
@@ -32,20 +35,39 @@ interface UploadFileParameter {
 	driveKey?: DriveKey;
 }
 
+type FilePath = string;
+
+const formatResults = (results: ArFSResult[]): ArFSResult => {
```

Comment:
Not critical, but a set of unit tests for this could be nice. I'd approve without it though.